Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM crystallang/crystal:1.9.2-alpine as Builder
+
+# install packages required to run the representer
+COPY . .
+
+RUN shards install
+
+RUN apk add --no-cache bash coreutils shards yaml-dev musl-dev make
```

Comment:
You can improve the Docker layer caching a bit by doing:

```suggestion
RUN apk add --no-cache bash coreutils shards yaml-dev musl-dev make

# install packages required to run the representer
COPY shard.lock shard.yml ./

RUN shards install

COPY . .
```

The first command is unlikeliest to change, so run that first. Then _only_ copy the shard files to install the dependencies, followed by the build.